client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -160,7 +160,7 @@ export const useCommandsArrayWpcom = ( {
 				_x( 'wp-admin', 'Keyword for the Open site dashboard command' ),
 			].join( ' ' ),
 			context: [ '/sites' ],
-			callback: setStateCallback( 'openSiteDashboard', __( 'Select dashboard to open' ) ),
+			callback: setStateCallback( 'openSiteDashboard', __( 'Select site dashboard to open' ) ),
```

Comment:
Should it be 'Select site to open dashboard'?

Reply:
Updated with 9e776ef122a